packages/@glimmer/debug-util/lib/simple-cast.ts

```diff
@@ -23,13 +23,14 @@ type BrowserTag = keyof BrowserTags;
 
 type NodeCheck<N extends Node> = (node: Node) => node is N;
 type SugaryNodeCheck<K extends BrowserTag = BrowserTag> = NodeCheck<BrowserTags[K]> | K | K[];
-type NodeForSugaryCheck<S extends SugaryNodeCheck<BrowserTag>> = S extends NodeCheck<infer N>
-  ? N
-  : S extends keyof BrowserTags
-    ? BrowserTags[S]
-    : S extends (keyof BrowserTags)[]
-      ? BrowserTags[S[number]]
-      : never;
+type NodeForSugaryCheck<S extends SugaryNodeCheck> =
```

Comment:
This is one of a number of places where the default value for a type parameter was explicitly
included. They were removed by autofix.